clients/extension/src/background/handlers/requestHandler.ts

```
@@ -16,6 +16,8 @@
   toUtf8String,
```

ESLint

[error] 'toUtf8String' is defined but never used.

(@typescript-eslint/no-unused-vars)